tests/integration/pilot/ingress_test.go

```diff
@@ -216,16 +216,21 @@ func TestIngress(t *testing.T) {
 				ingressutil.DeleteKubeSecret(t, []string{credName2})
 			})
 
+			apiVersion := "v1beta1"
+			if t.Clusters().Default().MinKubeVersion(19) {
+				apiVersion = "v1"
+			}
+
 			ingressClassConfig := `
-apiVersion: networking.k8s.io/v1beta1
+apiVersion: networking.k8s.io/%s
```

Comment:
Is "%s" a typo in the literal string "networking.k8s.io/%s"?

Reply:
no, we sprintf it later. I guess we could probably just do it here though